pipelines/folioRancher/folioDevTools/moduleDeployment/uninstallModules/Jenkinsfile

```diff
@@ -0,0 +1,105 @@
+#!groovy
+import org.folio.models.OkapiTenant
+import org.folio.models.RancherNamespace
+import org.folio.rest_v2.Main
+import org.folio.rest_v2.Authorization
+import org.folio.utilities.RestClient
+import org.jenkinsci.plugins.workflow.libs.Library
+import groovy.json.JsonSlurperClassic
+import groovy.json.JsonBuilder
+
+@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+
+properties([
+  buildDiscarder(logRotator(numToKeepStr: '30')),
+  disableConcurrentBuilds(),
+  parameters([
+    folioParameters.cluster(),
+    folioParameters.namespace(),
+    folioParameters.tenantId(''),
+    string(name: 'MODULE_NAME', defaultValue: '', description: '(REQUIRED) Module names separated by comma', trim: true),
+    folioParameters.agent(),
+    folioParameters.refreshParameters()
+    ])
+])
+
+def CLUSTER_URL = params.CLUSTER
+def NAMESPACE_URL = params.NAMESPACE
+def TENANT_URL = params.TENANT_ID
+
+if (params.REFRESH_PARAMETERS) {
```

Comment:
I think this code should be after properties block before other code

Reply:
done